packages/manager/apps/dedicated/client/app/account/identity-documents/user-identity-documents.html

```diff
@@ -0,0 +1 @@
+<h2 data-translate="user_account_identity_documents"></h2>
```

Comment:
may be having p with h2 style is better ?
```suggestion
<p class="oui-heading_2" data-translate="user_account_identity_documents"></p>
```